hadoop-tools/hadoop-azure/src/test/java/org/apache/hadoop/fs/azurebfs/ITestAzureBlobFileSystemAuthorization.java

```diff
@@ -18,20 +18,23 @@
 
 package org.apache.hadoop.fs.azurebfs;
 
+import java.io.*;
 import java.util.Arrays;
 import java.util.List;
 import java.util.UUID;
 
+import org.apache.hadoop.fs.azurebfs.constants.*;
```

Comment:
all into the org.apache.block. .* is only allowed for static imports..if that is what these are then move down there as import static entries. Otherwise: expand

Reply:
Fixed